website/index.html

```diff
@@ -0,0 +1,133 @@
+                <img class="deco-card" id="deco-card-2" src="../assets/textures/cards/explosiveBullet.png" /> 
+                <img class="deco-card" id="deco-card-3" src="../assets/textures/cards/rottenBullet.png" /> 
+                <img id="main-logo" src="./assets/logo_black.png" />
+                <a href="#" class="download">Play now!</a>
+                <img id="deco-revolver-1" src="./assets/revolver.png" />
+            </article>
+            <article id="trailer-container">
+                <img src="./assets/group_picture.png" />
+                <!-- <iframe
+                    src="https://www.youtube.com/embed/dQw4w9WgXcQ"
+                    title="YouTube video player"
+                    frameborder="0"
+                    allow="clipboard-write; encrypted-media; picture-in-picture; web-share"
+                    allowfullscreen
+                ></iframe> -->
+            </article>
+            <article class="paper-text-box" id="main-description">
+                <img src="assets/hat.png" id="deco-hat" />
+                <img class="deco-card" id="deco-card-4" src="../assets/textures/cards/workerBullet.png" /> 
+                <img class="deco-card" id="deco-card-5" src="../assets/textures/cards/leadersBullet.png" /> 
+                <img class="deco-card" id="deco-card-6" src="../assets/textures/cards/incendiaryBullet.png" /> 
+                <img class="deco-card" id="deco-card-7" src="../assets/textures/cards/explosiveBullet.png" /> 
+                <img class="deco-card" id="deco-card-8" src="../assets/textures/cards/bewitchedBullet.png" />
+                <div id="micro-comic-container">
+                    <img id="deco-micro-comic" src="./assets/micro_comic_sprint_planning.png">
+                </div>
+                <p class="cur-page-dependent">
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Odio voluptatem totam molestias modi necessitatibus ratione earum? Quaerat fugiat blanditiis consequatur iusto incidunt eligendi similique, earum illum, minima est debitis dicta.
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Odio voluptatem totam molestias modi necessitatibus ratione earum? Quaerat fugiat blanditiis consequatur iusto incidunt eligendi similique, earum illum, minima est debitis dicta.
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Odio voluptatem totam molestias modi necessitatibus ratione earum? Quaerat fugiat blanditiis consequatur iusto incidunt eligendi similique, earum illum, minima est debitis dicta.
+                </p>
+            </article>
+            <footer>
+                <div class="links">
+                    <a class="to-imprint-page">IMPRINT</a>
+                    <a href="#">SUPPORT US</a>
+                    <a href="#">GITHUB</a>
+                </div>
+                <div class="copyright">Copyright © 2023 (Name)</div>
```

Comment:
name missing, prob bec we don't have one

Reply:
placeholder